lib/CMakeLists.txt

```diff
@@ -454,6 +454,7 @@ message(STATUS "Loading version ${VERSION} into constants...")
 
 #double escape for windows backslash path separators
 string(REPLACE "\\" "\\\\" prefix "${prefix}")
+string(REPLACE "${CMAKE_SOURCE_DIR}" "$BUILD_DIR" COMPILER_INFO "${COMPILER_INFO}")
```

Comment:
is the 2nd argument intentionally not `${}` encapsulated? I don't think this matters for CMake, but it's worth asking as I'm not sure what the end result of this command will be for various SOURCE and BUILD options.